### PR TITLE
Upgrade to Lotus v1.14.2

### PR DESCRIPTION
**Pull request by Anonyma**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,7 +10,7 @@ require (
 	github.com/filecoin-project/go-fil-markets v1.19.0
 	github.com/filecoin-project/go-jsonrpc v0.1.5
 	github.com/filecoin-project/go-state-types v0.1.3
-	github.com/filecoin-project/lotus v1.14.0
+	github.com/filecoin-project/lotus v1.14.2
```
Comment (jleni) [inline go.mod:13]: are we upgrading to 1.14.2?
Does the PR title need a correction?

File: go.mod [hunk 2]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/multiformats/go-multihash v0.1.0
 	github.com/orcaman/concurrent-map v0.0.0-20190826125027-8c72a8bb44f6
 	github.com/stretchr/testify v1.7.0
-	github.com/zondax/rosetta-filecoin-lib v1.1400.1
+	github.com/zondax/rosetta-filecoin-lib v1.1401.0
```
Comment (jleni) [inline go.mod:24]: should we release a 1.1402.0?

**Review by jleni — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (Anonyma) [reply go.mod:24]: Yep, opened a PR for that

File: go.mod (hunk 1 above)
Comment (Anonyma) [reply go.mod:13]: Done

**Review by raynaudoe — APPROVED** [review]

**Review by jleni — APPROVED** [review]

**Pull request merged**